public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -131,7 +137,22 @@ export function DataGrid(props: DataGridProps) {
       return {
         visibleColumns: columns,
         setVisibleColumns: (visibleColumns: string[]) => {
-          // TODO: implement with sidebar field order (dragability) changes
+          if (explorerFields.selectedFields.length > 0) {
+            const fields: IField[] = [];
+            // fields within visibleColumns are re-created to be of type IField
+            visibleColumns.map((col) => {
```

Comment:
Should use forEach as you are not using the newly generated array at all

Reply:
changed to forEach